### PR TITLE
Moved Galileo version to win8, added win10 version

### PR DESCRIPTION
**Pull request by turkycat**

### CODE REVIEW

File: win10/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+#TODO Sample on Galileo
```
Comment (ooeygui) [inline win10/README.md]: This says todo sample on Galileo... Did you mean on Pi2?

File: win10/TodoSample/Package.appxmanifest [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:iot="http://schemas.microsoft.com/appx/manifest/iot/windows10"
+  IgnorableNamespaces="uap mp iot">
+
+  <Identity
+    Name="TodoSample-uwp"
+    Publisher="CN=jessefr"
```
Comment (ooeygui) [inline win10/TodoSample/Package.appxmanifest]: Microsoft

File: win10/TodoSample/Package.appxmanifest [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    Version="1.0.0.0" />
+
+  <mp:PhoneIdentity PhoneProductId="fc7a4124-9fbd-47ee-b67c-38ba2692713f" PhonePublisherId="00000000-0000-0000-0000-000000000000"/>
+
+  <Properties>
+    <DisplayName>TodoSample</DisplayName>
+    <PublisherDisplayName>jessefr</PublisherDisplayName>
```
Comment (ooeygui) [inline win10/TodoSample/Package.appxmanifest]: Microsoft

**Comment by ooeygui** [comment]
:-1:

**Comment by turkycat** [comment]
I've fixed the manifest and readme issues, thanks for pointing these out! Please review once more

**Pull request merged**